src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileRetrieverTest.java

```diff
@@ -133,7 +134,7 @@ public void should_throw_exception_when_query_throws_exception() {
 
     @Test
     public void should_retrieve_Multiple_Profiles() {
-        idamRolesInfo = new IdamRolesInfo(entity, HttpStatus.CREATED);
+        idamRolesInfo = new IdamRolesInfo(entity);
 
         List<UserProfile> userProfiles = new ArrayList<>();
 
```

Comment:
prefix all the test classes with test in this class

Reply:
Its not mandatory that test case name should start with Test or is it Hmcts mandate ?

Reply:
before adding comments please you go through the email QA assurance: John review comments and let comeback on this.